README.md

```diff
@@ -36,12 +36,14 @@ make check
 sudo make install
 ```
 
-Then, `go get -u` as usual.
+Then, `go get -u` as usual the following packages:
 
 ```sh
+go get -u github.com/golang/protobuf/protoc-gen-go
```

Comment:
I accept the first one, for the others added (grpc,context) I would prefer to tell users to 'go get .' after generating code.